index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    const push = this.items.push(item);
+    const ascending = this.items.sort((a,b) => a - b);
+    this.length++;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (!this.items[pos]){
+      throw new Error('OutOfBounds');
+    } else{
+     return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if(this.length === 0){
+      throw new Error('EmptySortedList');
+    }else{
+      return Math.max.apply(null, this.items);
+    }
+  }
 
-  min() {}
+  min() {
+    if(this.length === 0){
+      throw new Error('EmptySortedList');
+    }else{
+      return this.items[0];
+    }
+  }
 
-  sum() {}
+  sum() {
+    if(this.length === 0){
+      return 0;
+    }else{
+      return this.items.reduce((a, b) => a+b);
+    }
+   
+  }
 
-  avg() {}
+  avg() {
+    if(this.length === 0){
+      throw new Error('EmptySortedList');
+    }else{
+      let soma =this.items.reduce((a, b) => a+b);
+      return soma/this.length;
```

Comment:
Aqui, ao invés de fazer de novo a soma, você pode chamar a função anterior e fazer `this.sum()/this.length`